core/src/main/java/io/questdb/cairo/SymbolMapWriter.java

```diff
@@ -408,9 +406,36 @@ private int calculateMaxHashFromCapacity() {
         return Math.max(Numbers.ceilPow2(symbolCapacity / 2) - 1, 1);
     }
 
+    private void closeNoTruncate() {
+        // if we fail to rebuild files, we need to close them without truncates
+        // the files are links to previous version of symbol map
```

Comment:
nit: "a previous"

Reply:
done in [203603a](https://github.com/questdb/questdb/pull/5598/commits/203603aa7c248ccd9e4dcc322b7c31f8f642684d)